src/menu.c

```diff
@@ -458,12 +458,12 @@ void DisplayItemMessageOnField(u8 taskId, const u8 *string, TaskFunc callback)
     CopyWindowToVram(0, 3);
 }
 
-void DisplayYesNoMenu(void)
+void DisplayYesNoMenuWithDefault(void)
```

Comment:
I think these two names are swapped.  This one should be "default Yes".  The next one should be "with default" since the initial cursor position is an argument.